### PR TITLE
Tb/doc edits 

### PR DESCRIPTION
**Pull request by timantmedia**

### CODE REVIEW

**Comment by timantmedia** [comment]
@mustafaboleken Do you know why this failed in Travis? I cannot figure it out.

**Comment by timantmedia** [comment]
@mustafaboleken The travis PR listed here seems to be successful https://app.travis-ci.com/github/ant-media/Ant-Media-Server/pull_requests

**Comment by mustafaboleken** [comment]
> @mustafaboleken Do you know why this failed in Travis? I cannot figure it out.

I just get updates from master, if it fails again, I will check and tell you the reason @timantmedia

File: doc/website/docs/quick-start.md [hunk 1]
```diff
@@ -26,23 +25,18 @@ You can download Community Edition from [Releases](https://github.com/ant-media
...
 ```
 
 Get the `install_ant-media-server.sh` script.
 
-
-
 ```shell
-wget https://raw.githubusercontent.com/ant-media/Scripts/master/install_ant-media-server.sh 
-chmod 755 install_ant-media-server.sh
+wget https://raw.githubusercontent.com/ant-media/Scripts/master/install_ant-media-server.sh & chmod 755 install_ant-media-server.sh
```
Comment (asharamseervi) [inline doc/website/docs/quick-start.md:35]: In shell script, it's logical && operator, not just single &.
```wget https://raw.githubusercontent.com/ant-media/Scripts/master/install_ant-media-server.sh && chmod 755 install_ant-media-server.sh```

**Comment by mustafaboleken** [comment]
@timantmedia one of the test failed but it's not related with you, we can ignore this one

File: doc/website/docs/quick-start.md (hunk 1 above)
Comment (mustafaboleken) [reply doc/website/docs/quick-start.md:35]: you are right, we should use double ampersand

**Comment by mekya** [comment]
I've fixed the failing test and update this branch. So it's passing and merging for you. 

Btw, what's Tb/doc means? @timantmedia

**Review by mekya — APPROVED** [review]

**Comment by timantmedia** [comment]
> I've fixed the failing test and update this branch. So it's passing and merging for you.
> 
> Btw, what's Tb/doc means? @timantmedia

@mekya I just created my own branch using the format initials Tb (Tim Bobker) and then the reason for the branch (docs) just so I can find it and list all my branches if I create another one.

**Pull request merged**